.gitignore

```diff
@@ -52,6 +52,7 @@ cmake-build-*/
 
 # IntelliJ
 out/
+*.iml
```

Comment:
can you add `idea/encodings.xml` as well?

Reply:
So now we know that this breaks TeamCity builds. We have enough info to report to JetBrains or update our existing report. Do we have one in progress?